views/beers.hbs

```diff
@@ -0,0 +1,10 @@
+<div>
+<section class="beers">
+    <ul>
+        {{#each responseFromAPI}}
```

Comment:
quick piece of advice: responseFromAPI might not be the best name. Maybe beersList, or another name that you can immediately know what you are looking at if you are looking at code in a week and a half from now